themes/geekboot/layouts/partials/feature-state-alert.html

```diff
@@ -15,7 +15,7 @@
     <div class="bd-content mt-3">
         <p>
             Crossplane may change or drop this feature at any time. <br />
-            For more information read the <a href="https://docs.crossplane.io/latest/reference/feature-lifecycle/">Crossplane feature lifecycle</a>.
+            For more information read the <a href="{{.Site.BaseURL}}knowledge-base/guides/feature-lifecycle/" >}}'>Crossplane feature lifecycle</a>.
```

Comment:
This page is no longer under a version but part of the KB. 

`{{.Site.BaseURL}}` ensures the link works for both prod, PRs and local dev.